Cpix/ContentKeyPeriod.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Axinom.Cpix
+{
+	public sealed class ContentKeyPeriod : Entity
+	{
+		/// <summary>
+		/// Gets or sets the ID of the content key period. This must be unique
+		/// within the scope of this document and the value must be a valid XML
+		/// ID.
+		/// </summary>
+		public string Id { get; set; }
+
+		/// <summary>
+		/// Get or sets the numerical index for the key period. Mutually exclusive
+		/// with start and end.
+		/// </summary>
+		public int? Index { get; set; }
+
+		/// <summary>
+		/// Gets or sets the wall clock (Live) or media time (VOD) for the start time
+		/// for the period. Mutually inclusive with end, and mutually exclusive with
+		/// index.
+		/// </summary>
+		public DateTime? Start { get; set; }
```

Comment:
Is there a particular reason to use DateTime? DateTimeOffset is considered the more modern alternative.

DateTime has all kinds of edge cases and one would always need to convert all DateTimes to UTC to compare them. If the kind of a DateTime is not known, then comparisons will break.

For exampe:
`var dt1 = DateTime.Now;
var dt2 = DateTime.UtcNow;
Console.WriteLine(dt1 - dt2); // Almost 3 hours difference.`

With DateTimeOffset such ambiguities cannot occur.

Reply:
DateTime was used to accommodate "unspecified", which the XML datetime allows for and CPIX spec also make no mention that a timezone should be required. So, DateTime felt as the better fit ... in order not to lose any info (DateTimeOffset doesn't support an "unspecified" and considers it implictly as Local timezone).  

But upon further discussions, it seems that allowing unspecified timezones might be a recipe for confusion anyway and it might be better to consider all unspecified timezones as UTC -- it's at least constant, so handling doesn't depend any way on where the lib runs (different local timezone). Alternative would've been to consider input without timezone invalid outright -- but it's a trade-off between scaring users away with error messages or accepting some default.

So, I guess this is what we'll do. And in this case DateTimeOffset sounds fine. We can revisit this when we get more clarity from CPIX developers.